wdl2cwl/tests/test_cwl.py

```diff
@@ -8,6 +8,18 @@ def get_file(path: str) -> str:
     return os.path.join(os.path.dirname(__file__), path)
 
 
+@pytest.fixture(autouse=True)
+def mock_args(monkeypatch: pytest.MonkeyPatch, wdl_path: str) -> None:
+    """Patch sys.argv to all tests."""
```

Comment:
No, we don't want to always use the monkey patch :-)

Leave the first two tests alone. Write a third test that uses the MonkeyPatch trick.

Reply:
Is the new code fixing this?